src/pages/index.js

```diff
@@ -1,193 +1,216 @@
+
+        const cleanedText = cleanText(searchText)
+        const categoriesFlattened = categories
+            .filter(c => c.selected)
+            .map(c => c.title)
+            .join(", ")
+        if (cleanedText || categoriesFlattened) {
+            setActiveFilter(
+                `Text: ${cleanedText} | Categories: ${categoriesFlattened}`
+            )
+        } else {
+            setActiveFilter(null)
+        }
+    }, [elements, searchText, categories, activeFilter])
+
+    const debouncedActiveFilter = useDebounce(activeFilter, 1500)
+    useEffect(() => {
+        if (debouncedActiveFilter) {
+            trackCustomEvent({
+                category: "CESG",
+                action: "Search",
+                label: debouncedActiveFilter,
+            })
+        }
+    }, [debouncedActiveFilter])
+
+    return (
+        <Layout>
+            <SEO title="Gallery of Custom Element samples" />
+            <a href="https://kontent.ai" target="_blank" rel="noopener noreferrer">
+                <div id="logo"><img src="https://kontent.ai/img/general/kontent-by-kentico.svg"></img></div>
+            </a>
+            <section className="section grid">
+                <div className="grid__row">
+                    <div className="grid__col grid__col--12">
+                        <div className="js-pagination-list">
+                            <div className="heading heading--h2 heading--center heading--indent-content">
+                                <h2>
+                                    Gallery of Custom Element samples
+                                    <strong className="highlight">.</strong>
+                                </h2>
+                            </div>
+                            <div style={{display: 'flex', justifyContent: 'center'}}>
+                                <div className="callout callout--is-warning callout--has-s-shadow">
+                                    <div className="callout__title">
+                                        <i className="callout__title-icon icon-exclamation-triangle" />
+                                        Discontinuation notice
+                                    </div>
+                                    <div className="callout__message">
+                                        <p>
+                                            This website will be removed.
+                                            If you want to check out our core integrations,
+                                            please visit <a href="https://kontent.ai/integrations/">this page</a> or
+                                            check out <a href="https://github.com/topics/kontent-ai-custom-element">this topic</a> on GitHub.
+                                            If you want to make your integration visible,
+                                            use the <a href="https://github.com/topics/kontent-ai-custom-element">topic</a> in your repository.
```

Comment:
```suggestion
                                            use the <a href="https://github.com/topics/kontent-ai-custom-element">"kontent-ai-custom-element" topic</a> in your repository.
```

Or whatever name of the topic will be used - for easier copy and paste :-)